provisioner/terraform/testdata/rich-parameters/rich-parameters.tf

```diff
@@ -27,6 +27,23 @@ data "coder_parameter" "example" {
   }
 }
 
+// Plugin revision v0.7.0 requires defining "min" or "max" rules together.
+data "coder_parameter" "number_example_min_max" {
+  name    = "number_example_min_max"
+  type    = "number"
+  default = 4
+  validation {
+    min = 3
+    max = 6
```

Comment:
can we have a test case where min or max is explicitly 0?

Reply:
Added, although I had to refactor the logic a bit. "min" and "max" need to be specified together as it's a hard requirement by the `coder_parameter` schema